static/sass/mq.scss

```diff
@@ -159,4 +159,48 @@
  * for IE10 Snap Mode on Metro
  */
 @-ms-viewport { width: device-width; }
-@viewport { width: device-width; }
\ No newline at end of file
+@viewport { width: device-width; }
+
+
+
+@media(min-width: px2em(940px)) {
+  #mainnav .navigation.menu .tier-1 {
+    &.element-2 .subnav {
+      overflow: hidden;
+      overflow-y: scroll;
+      height: px2em(180px);
```

Comment:
I'm still not sure setting `height` to a fixed value is a good idea here. Don't we still see the original bug if we visit the website in a 1280x150px window?

Reply:
Yeah, Now it won't show original bug even the height is less than **180PX**.